db/migrate/20191020155822_add_phone_ciphertext_to_rebels.rb

```diff
@@ -5,6 +5,6 @@ def change
 
     # blind index
     add_column :rebels, :phone_bidx, :string
-    add_index :rebels, :phone_bidx, unique: true
```

Comment:
@mhulet If you add a migration to update the index, there is no need to update my migration. 
The easiest would be to keep the `add_phone_ciphertext_to_rebels.rb` update and remove the new migration `update_rebels_phone_bidx_uniqueness.rb`